main.js

```diff
@@ -76,7 +76,6 @@ document.querySelector('#Book-form').addEventListener('submit', (e) => {
   // prevent actual submit
   const title = document.querySelector('#title').value;
   const author = document.querySelector('#author').value;
```

Comment:
- [ ] I would recommend you to add little validation to your form, so that submitting empty inputs would be prevented. At this moment, title and author empty inputs can be submitted which is not too good for an efficient app. 
- [ ] After adding some books and trying to remove one book from your page, I noticed that clicking on one remove button removed all the books objects from your local storage and this behaviour is not what is demanded of this milestone. Below screen record is current behaviour of your app. Kindly look into it and fix where the issue is.

[your-page.webm](https://user-images.githubusercontent.com/26597432/204510926-855129f0-b32b-4d53-bbbe-04454790c96e.webm)

Reply:
Thank you for supportive comment and we will fix it